config/crd/bases/eda.ansible.com_edas.yaml

```diff
@@ -104,7 +105,8 @@ spec:
                     description: 'Size is the size of number of eda-api replicas.
                       Default: 1'
                     format: int32
-                    minimum: 0
+                    default: 1
```

Comment:
Should be the default "2" as we already do with the other workers?

Reply:
talked out of band and default of 1 is OK for now for API.  We can document setting it to 2 for redundancy if desired.